### ISSUE TITLE
Web site and singularity

### ISSUE THREAD

**Issue by andrewjmc** [kusterlab/prosit#11]
I am really excited to get started with Prosite, but it will take some time to get this working on HPC (singularity, no root access).

First question: any guidance on setting up with singularity?

Second question: on the website for calibration my file uploads are failing (both MSMS and Raw) -- I've disabled virus protection to no avail. When I open Chrome inspector I see an upload open but no data is transferred. I had no success in Firefox either. I have no issues with large file uploads to other sites (e.g. Dropbox). Any thoughts welcome.

Thanks for your help,

Andrew

**Comment by gessulat** [comment]
Hi Andrew,

1. Unfortunately we do not have any experience with singularity. Our docker file specifies all python dependencies needed. I would guess that the hard part will be the CUDA / GPU integration.

2. We see this issue internally as well, but it is very hard to reproduce and to pin down why this is happening. We are actively investigating this and are doing changes to the server right now. Please let us know if these issues persist later this week.

**Comment by tkschmidt** [comment]
@andrewjmc Could you please send me directly an email to tobias.k.schmidt at tum.de so we can talk about your second problem?

**Comment by andrewjmc** [comment]
Sure!

**Comment by MarcIsak** [comment]
Hi, 

I have also experienced some problems uploading the msms.txt file and the RAW-file for rescoring. Everytime the upload did either not start or it would be reset after uploading a tiny fraction of the data.
 It did not matter if I disabled the virus protection or not in Windows defender. 

But today for the first time, I could upload all data and submit the search. The big difference compared to previous times, was to change browser from Google Chrome to an older version of Mozilla Firefox. I did also disable the real-time virus protection ( like previous times when I failed). Maybe it can give you some clues on why things are problematic with the upload. 

A potential improvement to the uploading process could be to allow zipped files of the msms.txt and the RAW. 

Best,

Marc
